docs/using/selecting.md

```diff
@@ -16,27 +16,47 @@ This section provides details about the first.
 The Jupyter team maintains a set of Docker image definitions in the <https://github.com/jupyter/docker-stacks> GitHub repository.
 The following sections describe these images, including their contents, relationships, and versioning strategy.
 
+### jupyter/base-jupyter
+
+[Source on GitHub](https://github.com/jupyter/docker-stacks/tree/main/base-jupyter) |
+[Dockerfile commit history](https://github.com/jupyter/docker-stacks/commits/main/base-jupyter/Dockerfile) |
+[Docker Hub image tags](https://hub.docker.com/r/jupyter/base-jupyter/tags/)
+
+`jupyter/base-jupyter` is a small image supporting a majority of [options common across all core stacks](common.md).
+It is the basis for all other stacks on which Jupyter-related applications can be built (e.g., kernel-based containers,
+[nbclient](https://github.com/jupyter/nbclient) applications, etc.). As such, it does not
+contain application-level software like Jupyter Notebook server, Jupyter Lab or Jupyter Hub.
+
+It contains:
+
+- [Miniforge](https://github.com/conda-forge/miniforge) Python 3.x in `/opt/conda` with two package managers
+  - [conda](https://github.com/conda/conda): "cross-platform, language-agnostic binary package manager".
+  - [mamba](https://github.com/mamba-org/mamba): "reimplementation of the conda package manager in C++". We use this package manager by default when installing packages.
+- Unprivileged user `jovyan` (`uid=1000`, configurable, [see options in the common features section](./common.md) of this documentation) in group `users` (`gid=100`)
+  with ownership over the `/home/jovyan` and `/opt/conda` paths
+- `tini` as the container entrypoint
+- A `start.sh` script as the default command - useful for running alternative commands in the container as applications are added (e.g. `ipython`, `jupyter kernelgateway`, `jupyter lab`)
+- Options for a self-signed HTTPS certificate and passwordless sudo
+
 ### jupyter/base-notebook
 
 [Source on GitHub](https://github.com/jupyter/docker-stacks/tree/main/base-notebook) |
 [Dockerfile commit history](https://github.com/jupyter/docker-stacks/commits/main/base-notebook/Dockerfile) |
 [Docker Hub image tags](https://hub.docker.com/r/jupyter/base-notebook/tags/)
 
-`jupyter/base-notebook` is a small image supporting the [options common across all core stacks](common.md).
-It is the basis for all other stacks and contains:
+`jupyter/base-notebook` adds base Jupyter server applications like Notebook, Jupyter Lab and Jupyter Hub
+and serves as the basis for all other stacks besides `jupyter/base-jupyter`.
+
+It contains:
 
+- Everything in `jupyter/base-jupyter`
 - Minimally-functional Jupyter Notebook server (e.g., no LaTeX support for saving notebooks as PDFs)
 - [Miniforge](https://github.com/conda-forge/miniforge) Python 3.x in `/opt/conda` with two package managers
```

Comment:
This should be removed from here (and fixed in `base-jupyter` description).

Reply:
I'm not following this suggestion.  All other derived images include an "Everything in _predecessor image_" statement.  What are you suggesting be "_fixed in `base-jupyter` description_"?  I've removed the `Miniforge` stuff from both locations and only have the "package managers" listed on `base-jupyter`.

Reply:
Now it looks good 👍  Before, it was mentioned twice, and was two times wrong.